utils/scripts/ocm/templates/dms.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
perhaps you should specify in the name that this is for RHOAM, since RHODS uses deadmansnitch as well.

Reply:
I thought it wasn't necessary because we don't use RHODS directly in our script and besides the dms report as metadata the name which refers to rhoam. However, I can change this in a next PR